specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json

```diff
@@ -41,6 +41,9 @@
           }
         },
         "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
         "x-ms-examples": {
```

Comment:
shouldn't `Configurations_ListByResourceGroup` be marked the same way then? 🙂  Note that this is an SDK breaking change, but I assume nothing has been released so far?

Reply:
Definitly. @Prasanna-Padmanabhan please follow @olydis guidelines. Thanks!!

Reply:
The resource group will only contain one configuration. The subscription can have one configuration for itself and one for each of its resource groups, hence the need for paging.

Reply:
If only one page is expected from that operation, you can mark it with
``` JSON
   "x-ms-pageable": {
          "nextLinkName": null
   },
```
The UX of the generated SDK will be better, as the useless nesting `value` property of the return value is hidden from the user.

Reply:
I see, OK, added that decoration per your suggestion.